### ISSUE TITLE
OpenRazer support

### ISSUE THREAD

**Issue by stijnb1234** [Artemis-RGB/Artemis#830]
Could you add support for OpenRazer so that Razer devices can be controlled on Linux?

**Comment by RobertBeekman** [comment]
Hey, have you given OpenRGB a shot? You can use that in combination with Artemis. 

https://wiki.artemis-rgb.com/en/guides/user/devices/open-rgb

**Comment by stijnb1234** [comment]
> Hey, have you given OpenRGB a shot? You can use that in combination with Artemis.
> 
> https://wiki.artemis-rgb.com/en/guides/user/devices/open-rgb

Yes, but sadly my Razer keyboard has problems in Direct mode and with external control. It just turns off.
